### ISSUE TITLE
Main menu

### ISSUE THREAD

**Issue by crabctrl** [FRC2706/giorno#15]

**Comment by crabctrl** [comment]
@aadiamazing5 have you done any work on this? I'm happy to help if you don't know where to start

**Comment by aadiamazing5** [comment]
Yeah I have started but im still learning the basics of AS so the layout is there but the buttons done actually do anything. I can push it later im out of town right now. But yeah I will definitely need help for it thanks :)
